constants_local.xml

```diff
@@ -188,6 +192,8 @@
             <pubkey/>
         </peer>
     </lookups>
+    <upper_seed/>
```

Comment:
What does upper_seed and lower_seed used for? Can put together with <seed>?

Reply:
The idea behind separating them is there because the lower_seed will send transactions to upper_seed.